### ISSUE TITLE
[Question] Need to make single test not run in parallel while running on multiple browsers

### ISSUE THREAD

**Issue by mrrinatino** [microsoft/playwright#17422]
Hi
My scenario is this. In the configuration file, I have it written to run tests in parallel in three browsers by default.
But I have tests that need to be run sequentially in each of the browsers, otherwise, actions will be overwritten. How can I write in a test to run it consistently across browsers?
test.describe.serial doesn't work here, playwright still runs it in three browsers at once.

**Comment by mrrinatino** [comment]
Sorry, find question #14419 
Maybe there is an update?

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

Thank you for your involvement. This issue was closed due to limited engagement (upvotes/activity), lack of recent activity, and insufficient actionability. To maintain a manageable database, we prioritize issues based on these factors.

If you disagree with this closure, please open a new issue and reference this one. More support or clarity on its necessity may prompt a review. Your understanding and cooperation are appreciated.
